src/frontend/admin/src/components/presentational/dnd/DndDefaultRow.tsx

```diff
@@ -8,7 +8,7 @@ import DeleteOutlineRoundedIcon from "@mui/icons-material/DeleteOutlineRounded";
 import ModeEditOutlineTwoToneIcon from "@mui/icons-material/ModeEditOutlineTwoTone";
```

Comment:
In the commit [7f3bc0](https://github.com/openfun/joanie/commit/7f3bc0a2ecf8a77da27a9b17d29e8101912298c7) message :
```diff
-Currently, we cannot see the prices to which the products are attached. So we
+Currently, we cannot see the courses to which the product is attached. So we
```